airflow/providers/tabular/example_dags/example_tabular.py

```diff
@@ -0,0 +1,39 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+from datetime import timedelta
+
+from airflow import DAG
+from airflow.operators.bash import BashOperator
+from airflow.providers.tabular.hooks.tabular import TabularHook
+
+default_args = {
+    "owner": "airflow",
+    "depends_on_past": False,
+    "email": ["airflow@airflow.com"],
+    "email_on_failure": False,
+    "email_on_retry": False,
+}
+
+dag = DAG("tabular", default_args=default_args, schedule_interval=timedelta(1))
+
+t1 = BashOperator(
+    task_id="templated",
+    bash_command="env",
+    env={"TOKEN": TabularHook().get_token()},
+    dag=dag,
+)
```

Comment:
Would you mind following `with DAG(...)` convention that seems to prevail in example dags?

Reply:
Good call! Let me update this 👍🏻